playbooks/pre_checks.yml

```diff
@@ -0,0 +1,39 @@
+- name: Pre-Migration Checks
```

Comment:
I think it could be nice to add this into the prelude/tasks/main.yml role and make the prelude role as dependency of the conversion_host role.
In meta.yml just by adding:

`dependencies:
  - role: os_migrate.vmware_migration_kit.prelude`
  
  So the checks are automatically include and executed prior to conversion_host tasks.